packages/node/test/e2e/requester-fulfill.feature.ts

```diff
@@ -57,6 +45,7 @@ it('should call fail function on AirnodeRrp contract and emit FailedRequest if r
 
   const preInvokelogNames = await fetchAllLogNames(provider, deployment.contracts.AirnodeRrp);
   expect(preInvokelogNames).toEqual([
+    'SetSponsorshipStatus', // RrpRequester constructor
```

Comment:
doh! in this [PR](https://github.com/api3dao/airnode/pull/567) I missed to replace this fixed array with `expect.arrayContaining(postInvokeExpectedLogs)` 🤦🏻‍♂️ 

Since I added the missing `SetSponsorshipStatus` event the test is passing anyways but we should change it to be consistent with the rest of the code I think.

Reply:
Good point. I noticed this while fixing the test but then I forgot about it.

I will create a new PR for this.

Reply:
I have already changed it in the other branch if you like I'll push them now

Reply:
https://github.com/api3dao/airnode/pull/567/commits/cd501b2a963de1f7c74d0238e5b59fae14c4b6a7